### PR TITLE
Julia talk.

### PR DESCRIPTION
**Pull request by elliottbiondo**

### CODE REVIEW

File: _posts/2015-06-05-Julia.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Julia Programming Language
+author: Claudia Solis-Lemus
+time: 12-1 PM
+location: 3425 Sterling Hall
```
Comment (gonuke) [inline _posts/2015-06-05-Julia.md]: wrong address

**Comment by elliottbiondo** [comment]
Fixed.

**Comment by elliottbiondo** [comment]
Ping @gonuke.

**Comment by gonuke** [comment]
Thanks @elliottbiondo

**Pull request merged**